config-docker.js

```diff
@@ -0,0 +1,44 @@
+var config = {};
```

Comment:
Maybe this file could be renamed to `docker-config.js` so all the Docker related files are close when listing the files in the root repo directory alphabetically... :)